test/e2e/tsconfig.json

```
@@ -20,5 +15,5 @@
       "@datadog/browser-core": ["../../packages/core/src"]
     }
   },
-  "include": ["./scenario/**/*.ts", "./lib/**/*.ts", "./*.d.ts"]
+  "include": ["./*.js", "./**/*.js", "./scenario/**/*.ts", "./lib/**/*.ts", "./*.d.ts"]
```

Could this be simplified in:
```suggestion
  "include": ["."]
```
or even remove the "include"?

---

Removing it seems to work fine :)